libraries/AP_HAL_ChibiOS/hwdef/include/minimal_useful.inc

```diff
@@ -0,0 +1,99 @@
+# a set of minimal features on smaller boards that still includes useful things that people ask for
+# in particular RC telemetry, VTX and OSD control
+
+# disable GPS and airspeed backends
+include minimal.inc
+include save_some_flash.inc
+
+# these are not on by default on 1Mb boards but get asked for a lot
+define AP_TRAMP_ENABLED 1
+define HAL_CRSF_TELEM_TEXT_SELECTION_ENABLED 1
+
+# disable emitting nice strings when activating RC channel aux functions:
+define AP_RC_CHANNEL_AUX_FUNCTION_STRINGS_ENABLED 0
+
+# disable precision landing
+define PRECISION_LANDING 0
+
+# low-end boards aren't expected to be used in environments where
+# things like satellite communications are required:
+define HAL_HIGH_LATENCY2_ENABLED 0
+
+# Gripper isn't a vital feature for smaller boards
+define AP_GRIPPER_ENABLED 0
+
+# Sprayer isn't a vital feature for smaller boards
+define HAL_SPRAYER_ENABLED 0
+
+# disable use of onboard FFT library:
+define HAL_GYROFFT_ENABLED 0
+
+# several notify backends are rare and not supported by default on smaller boards:
+define AP_NOTIFY_NCP5623_ENABLED 0
+
+# Soaring is off by defauult on of smaller boards.
+define HAL_SOARING_ENABLED 0
```

Comment:
should not be removed from any F405 plane board!

Reply:
@Hwurzburg this is just copied from minimize_features, I'm almost certain we had a devcall conversation about how that was the right thing to do - so I think the only option is to move it back to minimize_features

Reply:
The problem is that soaring is huge - 8k

Reply:
ok I have dropped the esoteric battery backends and re-enabled soaring - only 6k left. But maybe ok for now.

Reply:
yes its expensive...so is quadplane...the main thing is that any current Plane user should not lose a major function except between major revisions and with adequate notification: that their glider will not longer thermal or their landing gear will stop working, etc.

Reply:
No-ones lost anything - this is about finding a smaller feature set that isn't as completely drastic as minimize_features currently is.

Reply:
if this does not impact existing boards, then including it or even quadplane is fine by me....it could be used on 30x30 boards with only 4-5 outputs since they would not be quadplanes nor probably used in thermaling gliders